src/server/services/data-transform.service.js

```diff
@@ -15,6 +15,15 @@ const transformAnswersForSummary = (cumulativeAnswers, addressLookups) => {
   delete data.supply_directly;
   delete data.supply_other;
 
+  data.business_import_export = transformBusinessImportExport(
+    data.directly_import,
+    data.directly_export,
+    data.no_import_export
```

Comment:
If you use my rewrite of the below code, remember to remove the third argument here